testutil/keeper/observer.go

```diff
@@ -99,18 +109,28 @@ func ObserverKeeperWithMocks(t testing.TB, mockOptions ObserverMockOptions) (*ke
 		sdkKeepers.ParamsKeeper.Subspace(types.ModuleName),
 		stakingKeeper,
 		slashingKeeper,
+		authorityKeeper,
 	)
 
 	k.SetParams(ctx, types.DefaultParams())
 
-	return k, ctx, sdkKeepers
+	return k, ctx, sdkKeepers, ZetaKeepers{
+		AuthorityKeeper: &authorityKeeperTmp,
```

Comment:
It is returned in this `ZetaKeepers` structure to be able to use the keeper outside of the expected keeper interface.
It will be independent from the mock, but in test using the mock, you should not use this structure